src/liquid/components/ld-tabs/ld-tabs.tsx

```diff
@@ -0,0 +1,97 @@
+import '../../components' // type definitions for type checks and intelliSense
+import { Component, Element, h, Host, Prop } from '@stencil/core'
+import { getClassNames } from '../../utils/getClassNames'
+
+let tabsCount = 0
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ */
+@Component({
+  tag: 'ld-tabs',
+  styleUrl: 'ld-tabs.css',
+  shadow: false,
+})
+export class LdTabs {
+  @Element() el: HTMLElement
+
+  /** Size of the tabs. */
+  @Prop() size?: 'sm' | 'lg'
+
+  /** Display mode. */
+  @Prop() mode?: 'ghost' | 'brand-color'
+
+  /** Sets border radii. */
+  @Prop() rounded?: 'all' | 'all-lg' | 'top' | 'top-lg'
+
+  private idDescriber = `ld-tabs-${tabsCount++}`
+
+  private updateTabs(currentLdTab) {
+    this.el
+      .querySelector('[aria-selected="true"]')
+      ?.closest('ld-tab')
+      .removeAttribute('selected')
+    currentLdTab.scrollIntoView({
+      behavior: 'smooth',
+      block: 'nearest',
+      inline: 'center',
+    })
+  }
+
+  private updateTabPanels(currentLdTab) {
+    const tabId = currentLdTab.querySelector('[role="tab"]').id
+    this.el.querySelectorAll('[role="tabpanel"]').forEach((tabpanel) => {
+      tabpanel.closest('ld-tabpanel').classList.add('ld-tabpanel--hidden')
+    })
```

Comment:
```suggestion
    this.el
      .querySelector('ld-tabpanel:not(.ld-tabpanel--hidden)')
      .classList.add('ld-tabpanel--hidden')
```

Reply:
I have just applied this with a slight modification (added ? before .classList).